### ISSUE TITLE
Ones with incorrect relative URL crashes lemmy

### ISSUE THREAD

**Issue by dessalines** [rileynull/RedditLemmyImporter#6]
```
99364 | Masterpost on the People's Republic of China | /r/GenZedong/comments/jldogq/masterpost_on_the_peoples_republic_of_china/ | `Originally posted on r/GenZhou by u/FaZe_salad (score +53)`  +|      43999 |        26990 | f       | f      | 2021-03-12 11:33:46 |         | f       | f    | f        |             |                   |            |               | http://changeme.ml/a7f53b7d97e048e426da773a | t
```

The URL column here is `/r/Genzedong...` . This is a relative url, and it crashes lemmy. It should be absolute. Alternately, parse all these into kotlin URL types before you add them. 

edit: So far, this was the only one I found with a URL like that.

**Comment by mesmere** [comment]
Hm, interesting, thanks. Yeah I'll take care of this.

**Comment by mesmere** [comment]
Fixed in 0.3
